parsing/pcoq.mli

```diff
@@ -34,6 +34,20 @@ module Entry : sig
   val name : 'a t -> string
 end
 
+module Lookahead : sig
+  type lookahead
```

Comment:
By convention, this should be called `t`.

Reply:
Right, fixed.